ddsp/training/docker/user_interaction.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+"""Script for gathering input, building the image and submitting job"""
+
+import os
+
+def get_input():
+  """Gathers input from user"""
+  # Determining path for data retrieval
+  input_dict = dict()
+  data_path = input('Insert a path for data retrieving. '
```

Comment:
in fact, you could do a general function for all for all of these prompts
`def prompt(arg, message, default):`

Where the default part of the message gets appended automatically, and